libs/sdk-ui-tests-e2e/.env.template

```diff
@@ -0,0 +1,5 @@
+HOST=staging3.intgdc.com
+USER_NAME=first.last@gooddata.com
+PASSWORD=
+AUTH_TOKEN=pgstg2
```

Comment:
Not sure, this template is just copypasted from KD. 

I have this in mind and I'm going to consult this with one of guys who were working on these tests for KD. It is not needed to run tests locally, but I'm keeping it in this template for now.

Reply:
KD is a private repo so having any kind of token set there is okayish, but this is a public repo so any token here is problematic. Please investigate what this token means in KD and consider it compromised and make sure it is invalidated if necessary.

Reply:
the token is needed because tests provision workspaces to ensure clean slate. in context of this repo, these secrets will have to come as part of env variables from jenkins